fastlane/lib/fastlane/actions/set_info_plist_value.rb

```diff
@@ -21,7 +21,8 @@ def self.run(params)
             plist[params[:key]] = params[:value]
           end
           new_plist = Plist::Emit.dump(plist)
-          File.write(path, new_plist)
+          output = params[:output] || path
```

Comment:
This might be inside a subfolder that doesn't exist yet, right? Maybe something we should add here? 
```ruby
FileUtils.mkdir_p(...)
```